.env.example

```diff
@@ -29,3 +29,6 @@ APP_PORT=8080
 # get this from the google analytics dashboard
 VUE_APP_GOOGLE_ANALYTICS_APP_NAME=
 VUE_APP_GOOGLE_ANALYTICS_ID=
+
+# mixpanel
+VUE_APP_MIXPANEL_PROJECT_TOKEN= # get this from the Mixpanel project dashboard
```

Comment:
Is there a plan to add the steps of this setup as a README? Even if the steps are very few?